.github/workflows/ci.yml

```
@@ -57,7 +57,10 @@ jobs:
           fetch-depth: 0
       - name: Install avocado libs
         working-directory: ./avocado-libs
-        run: pip install -e .
+        run: |
+         pip install --upgrade pip
+         pip install setuptools-rust
```

Looks that [setuptools-rust 1.1.2](https://pypi.org/project/setuptools-rust/1.1.2/) is the last version supports python3.6, therefore I'd suggest:
```suggestion
         pip install 'setuptools-rust==1.1.2'
```